concent_api/core/tests/utils.py

```diff
@@ -288,26 +295,32 @@ def _get_deserialized_task_to_compute(
         return task_to_compute
 
     def _get_deserialized_want_to_compute_task(
-        self,
-        timestamp: Union[str, datetime.datetime, None] = None,
+        self, timestamp: Union[str, datetime.datetime, None] = None,
         node_name: int = None,
         task_id: Optional[str] = None,
-        perf_index = None,
-        max_resource_size = None,
-        max_memory_size = None,
-        num_cores = None,
-        price = None,
-        concent_enabled = None,
-        extra_data = None,
+        perf_index=None,
+        max_resource_size=None,
+        max_memory_size=None,
+        num_cores=None,
+        price=None,
+        concent_enabled=None,
+        extra_data=None,
         provider_public_key: Optional[str] = None,
         provider_ethereum_public_key: Optional[str] = None,
+        requestor_private_key: Optional[bytes] = None,
     ) -> WantToComputeTask:
 
-        """ Returns WantToComputeTask deserialized. """
+        """ Returns WantToComputeTask deserialized.
+        :param requestor_public_key:
```

Comment:
This comment is not needed. Also, there should be no empty line between end of function signature and comment.